config.tpl.yml

```diff
@@ -225,6 +225,10 @@ slurm:
   # SLURM version to install. Currently supported: only 20.11.9 and 22.05.3.
   # Other versions can be installed by building from source (See build_rpms setting in the slurmserver role)
   slurm_version: 20.11.9
+  # Name of the SLURM cluster for accounting (optional, default to 'slurm')
+  # WARNING: changing this value on a running cluster will cause slurmctld to fail to start. This is a
+  # safety check to prevent accounting errors. To override, remove /var/spool/slurmd/clustername
+  # cluster_name: slurm
```

Comment:
As this is the template, I would set it. And I would change the cluster_name to something like slurm_azhop

Reply:
agree, and the name 'slurm' is too generic